scripts/explain_manifest/suites.py

```diff
@@ -30,7 +30,9 @@ def common_suites(expect, libxcrypt_no_obsolete_api: bool = False):
         .has_ngx_http_request_t_DW.equals(True)
 
     expect("/usr/local/openresty/nginx/sbin/nginx", "nginx binary should link pcre statically") \
-        .exported_symbols.contain("pcre_free") \
+        .exported_symbols.contain("pcre2_general_context_free_8") \
+        .needed_libraries.do_not().contain_match("libpcre2.so.+") \
```

Comment:
the dynamica library is named with `libpcre2-8.so.0` or `libpcre2-32.so.0`, so this
always evaluates to false, we might need something like
```
.needed_libraries.do_not().contain_match("libpcre2\-(8-32).so.+") \
```
or just
```
.needed_libraries.do_not().contain_match("libpcre.+.so.+") \
```
to match both pcre and pcre2

Reply:
I added them both.